.circleci/config.yml

```diff
@@ -56,7 +56,7 @@ run_tests: &run_tests
     - store_test_results:
           path: ~/rspec
     - store_artifacts:
-          path: tmp/capybara
```

Comment:
@damianlegawiec If you know how to get the screen shots available for failed tests in the Circle CI that would be useful for debugging those random fails, I couldn't do it.